CrowdStrikeFalcon/crowdstrike_falcon/custom_iocs.py

```diff
@@ -106,7 +106,10 @@ def get_valid_indicators(self, stix_objects):
     def run(self, arguments):
         if arguments.get("sekoia_base_url"):
             self.sekoia_base_url = arguments.get("sekoia_base_url")
-        stix_objects = arguments.get("stix_objects", [])
+        if "file_path" in arguments:
```

Comment:
The action never expects a `file_path` arguments. The action manifest should be updated to have:

* a `stix_objects_path` arguments
* Either `stix_objects` or `stix_objects_path` required

Then replacing `stix_objects = arguments.get("stix_objects", [])` with `stix_objects = self.json_argument("stix_objects", arguments)` will be enough to look inside the file or the argument.

The version in the manifest must also be bumped